src/index.js

```diff
@@ -49,6 +50,16 @@ function tileReduce(options) {
   return ee;
 }
 
-function bboxTiles(bbox, zoom) {
-  return tilecover.tiles(bboxPolygon(bbox).geometry, {min_zoom: zoom, max_zoom: zoom});
+function parseToTiles(options, zoom) {
+  var poly;
+
+  if (options.tiles) return options.tiles;
+
+  if (options.bbox) {
+    poly = bboxPolygon(options.bbox);
+  } else if (options.geojson) {
+    poly = options.geojson;
+  }
+
+  return tilecover.tiles(normalize(poly).features[0].geometry, {min_zoom: zoom, max_zoom: zoom});
```

Comment:
Instead of the normalize module, we could do something more lightweight (accepting a feature or a geometry) — tilecover would take care of the proper input validation:

```js
cover.tiles(geojson.geometry || geojson, ...)
```